src/tests/Myr_random.test.js

```diff
@@ -0,0 +1,125 @@
+import Myr from "../myr/Myr";
+
+const myr = new Myr();
+
+describe("Random Function Tests", () => {
+    
+    //randomInt
+    it("randomInt should move box to random x coordinate", () => {
+        let xPos = myr.randomInt();
+        myr.setPosition(xPos, 1, 1);
+        expect(myr.cursor.position).toEqual({ x: xPos, y: 1, z: 1 });
+    });

```

Comment:
What is this test actually testing? It's testing setPosition more than randomInt.